lib/internal/policy/sri.js

```diff
@@ -34,10 +34,10 @@ const parse = (str) => {
   const entries = [];
   while (match = RegExpPrototype.exec(kSRIPattern, str)) {
     if (match.index !== prevIndex) {
```

Comment:
Could it be one whole condition with below ?

Reply:
could be yea, i think readability is better as is but if requested to change I could